### ISSUE TITLE
Extend spec tests to support other browsers

### ISSUE THREAD

**Issue by ryanhofdotgov** [18F/e-QIP-prototype#954]
Want to be able to run the spec tests against our supported desktop browsers in addition to Chrome:
- [ ]  IE11
- [ ] Firefox
- [ ] Edge
- [ ] Safari on macOS

Integration with CircleCI or NBIS self-hosted CI is out of scope for this ticket and will be a follow-on.

**Comment by gidjin** [comment]
I sort of expect this to be dependent on #441 fixing the specs first

**Comment by ryanhofdotgov** [comment]
Yup, dependent on #441
